cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -138,6 +137,9 @@
   public static final String TOPIC_BY_REPLICATION_FACTOR = "topic_by_replication_factor";
   public static final String NO_REASON_PROVIDED = "No reason provided";
   public static final String DO_AS = "doAs";
+  public static final String BROKER_DIFF = "broker_diff";
+  public static final String PARTITION_COUNT = "partition_count";
+  public static final String TOPIC_NAME = "topic_name";
```

Comment:
There is already a `TOPIC_PARAM` for getting topic names.

Can we avoid introducing different parameter names for the same functionality and stick with the existing parameter name?